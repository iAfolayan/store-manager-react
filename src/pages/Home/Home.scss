.store-background {
	background-image: url('./images/bg2.jpg');
	background-position: center;
	background-size: cover;
	background-repeat: no-repeat;
	height: 100vh;
}