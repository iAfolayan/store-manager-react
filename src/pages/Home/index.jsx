import React from 'react';

function Home() {
  return <div className="home">hello</div>;
}

export default Home;
