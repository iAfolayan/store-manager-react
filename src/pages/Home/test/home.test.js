import React from 'react';
import { shallow } from 'enzyme';
import Home from '..';

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
