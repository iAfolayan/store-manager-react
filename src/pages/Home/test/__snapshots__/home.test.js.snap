// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div
  className="store-background"
>
  <FormContainer
    title="Store Manager"
  >
    <Connect(LoginForm) />
  </FormContainer>
  <Footer
    footerClassName="footer login"
  />
</div>
`;
